operations/helm/charts/mimir-distributed/README.md.gotmpl

```diff
@@ -120,67 +104,13 @@ enterprise:
 
-## Scale values
+## Sizing values
 
 The default Helm chart values in the `values.yaml` file are configured to allow you to quickly test out Grafana Mimir.
 Alternative values files are included to provide a more realistic configuration that should facilitate a certain level of ingest load.
 
-### Small
-
-The `small.yaml` values file configures the Grafana Mimir cluster to
-handle production ingestion of ~1M series using the blocks storage engine.
-Query requirements can vary dramatically depending on query rate and query
-ranges. The values here satisfy a "usual" query load as seen from our
-production clusters at this scale.
-It is important to ensure that you run no more than one ingester replica
-per node so that a single node failure does not cause data loss. Zone aware
-replication can be configured to ensure data replication spans availability
-zones. Refer to [Zone Aware Replication]({{ template "chart.homepage" . }}operators-guide/configuring/configuring-zone-aware-replication/)
-for more information.
-Minio is no longer enabled and you are encouraged to use your cloud providers
-object storage service for production deployments.
-
-To deploy a cluster using `small.yaml` values file:
-
-```bash
-helm install <cluster name> grafana/mimir-distributed -f small.yaml
-```
-
-### Large
-
-The `large.yaml` values file configures the Grafana Mimir cluster to
-handle production ingestion of ~10M series using the blocks storage engine.
-Query requirements can vary dramatically depending on query rate and query
-ranges. The values here satisfy a "usual" query load as seen from our
-production clusters at this scale.
-It is important to ensure that you run no more than one ingester replica
-per node so that a single node failure does not cause data loss. Zone aware
-replication can be configured to ensure data replication spans availability
-zones. Refer to [Zone Aware Replication]({{ template "chart.homepage" . }}operators-guide/configuring/configuring-zone-aware-replication/)
-for more information.
-Minio is no longer enabled and you are encouraged to use your cloud providers
-object storage service for production deployments.
-
-To deploy a cluster using the `large.yaml` values file:
-
-```bash
-helm install <cluster name> grafana/mimir-distributed -f large.yaml
-```
-
-# Development
-
-To configure a local default storage class for k3d:
-
-```bash
-kubectl apply -f https://raw.githubusercontent.com/rancher/local-path-provisioner/master/deploy/local-path-storage.yaml
-kubectl patch storageclass local-path -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
-```
-
-To install the chart with the values used in CI tests:
-
-```bash
-helm install test ./ --values ./ci/test-values.yaml
-```
+See [Capacity planning and Pod scheduling](https://grafana.com/docs/mimir/latest/operators-guide/run-production-environment-with-helm/guide/#capacity-planning-and-pod-scheduling)
+for more details about the sizing plans.
```

Comment:
```suggestion
For details about the sizing plans, see [Capacity planning and Pod scheduling](https://grafana.com/docs/mimir/latest/operators-guide/run-production-environment-with-helm/guide/#capacity-planning-and-pod-scheduling).
```

Reply:
Please check this link to see that it is working.

Reply:
I edited your comment and committed the change